playground/public/index.html

```diff
@@ -212,7 +210,10 @@ <h3 class="text-sm font-medium text-yellow-800">Browser compatibility warnings</
 
             const php = new PhpWeb({ sharedLibs, persist: [{ mountPath: '/persist' }, { mountPath: '/config' }] });
             await php.binary;
-            php.addEventListener('output', event => console.log(event.detail));
+            php.addEventListener('output', function (event) {
+                const status = JSON.parse(event.detail)
+                flavorLaunch.innerHTML = `Unpacking files... ${status.progress}`
+            });
             php.addEventListener('error', event => console.log(event.detail));
```

Comment:
per the video in the issue, the element shows it has been updated but it's not reflecting in the page